src/main/pages/che-6/user-guide/version-control.adoc

```diff
@@ -67,124 +97,127 @@ CHE_OAUTH_GITHUB_TOKENURI=https://github.com/login/oauth/access_token
 
 image::git/git-pull.png[]
 
-You can use `Rebase instead of merge` to keep your local commits on top (for more information https://git-scm.com/docs/git-pull#git-pull--r[see git docs]).
+Use `Rebase instead of merge` to keep your local commits on top. For more information, see the https://git-scm.com/docs/git-pull#git-pull--r[git docs].
 
-*Branches*
+=== Branches
 
-Manage your git branches at `Git` -> `Branches...` from main menu or `Ctrl + b` shortcut.
+Manage your Git branches using the *Git* > *Branches* menu options or use the `Ctrl + b` keyboard shortcut.
 
 image::git/git-branches-window.png[]
 
-You can filter branches view by choosing to see only local or remote branches.
+You can filter the branches view by choosing to see only local or remote branches.
 
-[id="compare"]
-== Compare
+[id="using-the-compare-window-to-view-changed-files"]
+== Using the Git Compare window to view changed files
 
-Git compare window is used to show files which was changed. This widget could be invoked from different places for different states, but the most frequently used way is to use `Git` -> `Compare` -> `<Select-to-what>` from main menu or use `Ctrl + Alt + D` shortcut to compare current state of code to the latest local commit. Another way is to select an object in project tree and click `Git` -> `Select-to-what` from item’s context menu. Note, this widget shows changed files in the selected object in project explorer. To see all changes select project folder. Note, if only one file was changed, diff window will be shown instead of compare window.
+The *Git Compare* window is used to show files that have been changed. You can invoke this widget from different places for different states. The most frequently used choice is the *Git* > *Compare* > *<Select-to-what>* menu options or the `Ctrl + Alt + D` keyboard shortcut to compare the current state of code with the latest local commit. Another option is to select an object in the project tree and click *Git* > *<Select-to-what>* from the context menu of the item. Note that this widget shows changed files in the selected object in the *Project Explorer*. To see all the changes select the project folder. If only one file was changed, the diff window is shown instead of the compare window.
 
 image::git/git-compare-tree-view.png[]
 
 By default, affected files are listed as a tree.
 
-`Expand all directories` and `Collapse all directories` will help you get a better view. `View as list` button switches the view of changed files to a list, where each file is shown with its full path. To return to the tree view just click `Group by directories`.
+The *Expand all directories* and *Collapse all directories* buttons show better views. The *View as list* button switches the view of changed files to a list, where each file is shown with its full path. To return to the tree view, click the *Group by directories* button.
```

Comment:
"Better" is subjective. By "better views", do we mean that there is more relevant information available to the user?